backend/src/models/billable.js

```diff
@@ -11,4 +11,26 @@ export class Billable {
       }
     });
   }
-}
+
+  deleteBillable(billableId, result) {
+    con.query("CALL delete_billable(?)", billableId, (err, res) => {
+      if (err) {
+        console.log("error: ", err);
+        result(err, null);
+      } else {
+        result(null, res[0]);
+      }
+    });
+  }
+
+  saveBillable(billable, result) {
+    con.query("CALL save_billable(?,?,?,?,?,?,?,?,?)", billable, (err, res) => {
```

Comment:
there's 10 params in save_billable, also the 2nd param in the con.query needs to be an array